src/compounds/carousel/src/index.tsx

```diff
@@ -0,0 +1,60 @@
+/** @jsx jsx */
+import React from 'react'
+import { jsx, useThemeUI } from 'theme-ui'
+import Glider from 'react-glider-carousel'
+
+import { gliderStyles } from './glider-styles'
+
+interface CarouselProps extends React.HTMLAttributes<HTMLDivElement> {
+  children: React.ReactElement<CarouselProps>[]
+  slidesMobile?: number
+  slidesTablet?: number
+  slidesDesktop?: number
```

Comment:
An array of device type values (like theme-ui supports) would be nice here.

```suggestion
  slides?: number[]
```